### PR TITLE
Adding timezone conversion for Events

### PR DESCRIPTION
**Pull request by chrisjaimes**
Short, easy to read UI PR: 

* Adding timezone conversion for Events
* Created Timezone dropdown component

Testeado 🚀 ✅

### CODE REVIEW

File: app/components/forms/EditEventForm.tsx [hunk 1]
```diff
@@ -131,7 +131,7 @@ const EditEventForm: React.FC<EditEventFormProps> = ({ event }) => {
         throw new Error(`${errorText}`);
       }
 
-      setSuccessMessage('Event created successfully');
+      setSuccessMessage('Event updated successfully');
```
Comment (chrisjaimes) [inline app/components/forms/EditEventForm.tsx:134]: These are just formatting changes below

File: app/components/timezonesDropdown.tsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
+import momentTZ from 'moment-timezone';
```
Comment (chrisjaimes) [inline app/components/timezonesDropdown.tsx:1]: New Component dropdown for timezones (similar to Countries dropdown component)

File: app/components/adminEventDetails.tsx [hunk 3]
```diff
@@ -319,22 +319,10 @@ const AdminEventDetails: React.FC<AdminEventDetailsProps> = ({
...
-                      {
-                        hour: '2-digit',
-                        minute: '2-digit',
-                        second: '2-digit',
-                      }
-                    )}
+                    {moment
+                      .utc(event.StartDateTimeUtc)
+                      .local()
+                      .format('MMM DD, YYYY h:mm A')}
```
Comment (chrisjaimes) [inline app/components/adminEventDetails.tsx:325]: ^ event start date conversion

File: app/components/adminPortal.tsx [hunk 4]
```diff
@@ -324,17 +324,10 @@ const AdminPortal: React.FC<AdminPortalProps> = ({ onSelectItem }) => {
...
-                          year: 'numeric',
-                          hour: 'numeric',
-                          minute: 'numeric',
-                          hour12: true,
-                        }
-                      )}
+                      {moment
+                        .utc(event.StartDateTimeUtc)
+                        .local()
+                        .format('MMM DD, YYYY h:mm A')}
```
Comment (chrisjaimes) [inline app/components/adminPortal.tsx:330]: ^ event start date conversion in admin portal

File: app/components/forms/EditEventForm.tsx [hunk 5]
```diff
@@ -438,10 +510,6 @@ const EditEventForm: React.FC<EditEventFormProps> = ({ event }) => {
             </div>
           </form>
         </div>
-        {/* Right Column Content */}
-        <div>
```
Comment (chrisjaimes) [inline app/components/forms/EditEventForm.tsx:442]: Unnecessary <div>

**Review by dnsantosuosso — APPROVED** [review]

**Pull request merged**